frontend/src/components/Admin/DevPortfolio/EditDevPortfolio.tsx

```diff
@@ -0,0 +1,52 @@
+import React, { useState } from 'react';
+import { Modal, Icon } from 'semantic-ui-react';
+import AdminDevPortfolioForm from './AdminDevPortfolioForm';
+import styles from './EditDevPortfolio.module.css';
+import DevPortfolioAPI from '../../../API/DevPortfolioAPI';
+import { Emitters } from '../../../utils';
+
+const EditDevPortfolio = (props: {
+  devPortfolio: DevPortfolio;
+  setDevPortfolios: React.Dispatch<React.SetStateAction<DevPortfolio[]>>;
+}): JSX.Element => {
+  const { devPortfolio, setDevPortfolios } = props;
+  const [open, setOpen] = useState(false);
+
+  const editDevPortfolio = (portfolio: DevPortfolio) => {
+    DevPortfolioAPI.updateDevPortfolio(portfolio).then((val) => {
+      if (val.error) {
```

Comment:
`val === undefined` is a sign that the DevPortfolio update did not go through and some error was thrown. As developers, we can inspect the network tab or look at the backend logs to see further what went wrong here. No need to expose the error message itself on the UI.